server/tests/DAL/report.dao.test.js

```diff
@@ -5,17 +5,22 @@ const { sequelize,
     checkPropertyExists
 } = require('sequelize-test-helpers');
 
-const [user, cr, crd, ReportTypeModel, RecipientModel, ReportTypeRecipientModel] = require('../../data_access_layer/models/localdb/localdb.model')(sequelize, dataTypes);
+const [user, cr, crd, PerformanceReportModel, ReportTypeModel, RecipientModel, ReportTypeRecipientModel] = require('../../data_access_layer/models/localdb/localdb.model')(sequelize, dataTypes);
 const ReportDAO = require('../../data_access_layer/daos/report.dao');
 
 
 describe("Test Report Related Models", () => {
     const ReportTypeInstance = new ReportTypeModel();
     const RecipientInstance = new RecipientModel();
+    const PerformanceInstance = new PerformanceReportModel();
+    checkModelName(PerformanceReportModel)('performance_reports');
     checkModelName(ReportTypeModel)('report_types');
     checkModelName(RecipientModel)('recipients');
     checkModelName(ReportTypeRecipientModel)('report_type_recipients');
 
+    ['performanceReportId', 'employeeId', 'averageCollectionDay', 'annualBillingObjective', 'monthlyBillingObjective', 'annualBillingNumber', 'monthlyBillingNumber', 'projectedBonus']
+        .forEach(checkPropertyExists(PerformanceInstance));
+
```

Comment:
This is repeated code from chart report dao test. So either put it in the report dao test or in the chart report dao test. However like I said in my other comment, I think the function to get performance reports should be in the report dao rather than the chart report dao.